setup

```diff
@@ -285,10 +282,11 @@ def build_and_copy_rust_modules(args: Dict[str, Any]) -> bool:
 
     logger.info(f"[Terminal] (7/8) Building and copying Rust modules started.")
 
-    os.chdir(RS_DIRECTORY)
-    for project in filter(
-        lambda f: os.path.isdir(f) and f != "rsmgp-sys", os.listdir(RS_DIRECTORY)
-    ):
+    for project in os.listdir(RS_DIRECTORY):
+        os.chdir(RS_DIRECTORY)
```

Comment:
os.chdir is needed in every iteration, because on line 293/291 we chdir there, and then the if becomes undeterministic.